src/module_3/train.py

```diff
@@ -0,0 +1,67 @@
+from pathlib import Path
+
+import pandas as pd
+import numpy as np
+import os
+import joblib
+
+from sklearn.linear_model import LogisticRegression
+
+
+from general_functions import prepare_data, train_val_split, standardise_data, evaluate_model
+
+# We only consider the features which we have seen previously that are relevant
+FEATURE_COLS = ['ordered_before', 'global_popularity', 'abandoned_before', 'normalised_price']
+TARGET_COL = 'outcome'
+Cs = [1e-8, 1e-4, 1e-2]
+
+OUTPUT_PATH = Path("/mnt/c/Users/Adriana/Desktop/ZRIVE/zrive-ds/src/module_3/out")
+DATA_PATH = Path("/mnt/c/Users/Adriana/Desktop/ZRIVE/data/groceries/box_builder_dataset/feature_frame.csv")
+
+
+def ridge_model_selection(data):
```

Comment:
A training script will always use all of its data. The split is done in the notebook to tune the hyperparemeters and have an understanding of the expected performance of the model. However, on your training script you want to leverage as much data as possible.

You can have another script dedicated to hyperpameter tuning that is a different step in the ETL and you may not necessary want to run it at the same pace than training.